integrations/mixpanel/lib/index.js

```diff
@@ -316,6 +317,32 @@ Mixpanel.prototype.alias = function(alias) {
   mp.alias(to, alias.from());
 };
 
+/**
+ *
+ * @param {Group} group
+ */
+
+Mixpanel.prototype.group = function(group) {
+  var groupIdentifierTraits = this.options.groupIdentifierTraits;
+  var groupId = group.groupId();
+  var userId = this.analytics.user().id();
+  var traits = group.properties();
+  if (!groupId || !userId || !groupIdentifierTraits.length) {
+    return;
+  }
+
+  if (traits && Object.keys(traits).length) {
+    for (var ind = 0; ind < groupIdentifierTraits.length; ind++) {
```

Comment:
Consistency fix for this PR, let's rename `ind` to `i`.

Reply:
Sadly we can't. In old versions of Javascript the for loop doesn't have its own scope. If we call it the same way, we'll be reusing the variable. It's a bad practice and the linter also blames me :D

Reply:
Ooh got it right.